File: app/features/document_intelligence/fixtures/simple.cpp
// Quarterly infrastructure report helper.

#include <string>

class RegionReport {
public:
    RegionReport(std::string name, int nodes, double utilization)
        : name_(std::move(name)), nodes_(nodes), utilization_(utilization) {}

    bool IsHealthy() const { return utilization_ < 0.85; }

private:
    std::string name_;
    int nodes_;
    double utilization_;
};

File: app/features/document_intelligence/fixtures/simple.py
"""Quarterly infrastructure report helper."""

import json


class RegionReport:
    """Summarizes capacity for a single region."""

    def __init__(self, name: str, nodes: int, utilization: float) -> None:
        self.name = name
        self.nodes = nodes
        self.utilization = utilization

    def is_healthy(self) -> bool:
        return self.utilization < 0.85


def load_report(path: str) -> list[RegionReport]:
    with open(path) as f:
        data = json.load(f)

    return [
        RegionReport(entry["region"], entry["nodes"], entry["utilization"])
        for entry in data
    ]
